.github/workflows/release.yml

```diff
@@ -0,0 +1,118 @@
+        required: true
+        type: boolean
+        description: Publish Releases at Anytime
+
+  workflow_run:
+    workflows: [ 🧪 Tests and Checks ]
+    branches: [main]
+    types: [ completed ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main' && github.repository_owner == 'wnfs-wg'
+
+    steps:
+      - name: Run release-please
+        id: release
+        uses: google-github-actions/release-please-action@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          default-branch: main
+          command: manifest
+          extra-files: |
+            Cargo.toml
+
+  publish-release-crates:
+    if: ${{ needs.release.outputs.releases_created || github.event.inputs.force-publish }}
+
+    runs-on: ubuntu-latest
+    needs: [ release-please ]
+
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Cache Project
+        uses: Swatinem/rust-cache@v2
+
+      - name: Install Rust Toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          override: true
+          profile: minimal
+          toolchain: stable
+
+      - name: Verify Publishing of crate
+        uses: katyo/publish-crates@v1
+        with:
+          dry-run: true
+
+      - name: Cargo Publish to crates.io
+        uses: katyo/publish-crates@v1
+        with:
+          registry-token: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+
+  publish-release-npm:
+    if: ${{ needs.release.outputs.releases_created || github.event.inputs.force-publish }}
+
+    runs-on: ubuntu-latest
+    needs: [ release-please ]
+
+    permissions:
+      contents: write
+      pull-requests: write
+
+    defaults:
+      run:
+        working-directory: ./wnfs-wasm
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Cache Project
+        uses: Swatinem/rust-cache@v2
+
+      - name: Install Rust Toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          override: true
+          profile: minimal
+          toolchain: stable
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install wasm-pack
+        run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+
+      # Builds output via wasm-pack and renames package.json name to remove
+      # `-wasm`.
+      - name: Build
+        run: |
+          wasm-pack build --target web --out-name wnfs
+          sed -i -e 's/"name": "wasm-wnfs"/"name": "wnfs",\n  "type": "module"/g' pkg/package.json
+
+      - name: Publish to npm
+        run: wasm-pack publish
+        working-directory: pkg
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
same as above: @walkah @appcypher @matheus23, we need to set this in secrets. For publishing, it's prob set to run w/ @matheus23's token, but we should solve this once and for all, though npm prob allows for multiple owners.

Reply:
Looking at npm, the npmjs "developers team" of the fission org has write access. Do we have an NPM auth token for that team? (Is that how that works? :P )

Reply:
I've added an org-level secret for `NPM_TOKEN` so this should work.